extension_helpers/_setup_helpers.py

```diff
@@ -85,6 +111,24 @@ def get_extensions(srcdir='.'):
                         [os.path.join(main_package_dir, '_compiler.c')])
         ext_modules.append(ext)
 
+    use_limited_api = False
+    abi = get_distutils_option('py_limited_api', ['bdist_wheel'])
+    if abi:
+        version_info = _abi_to_version_info(abi)
+        if version_info:
+            use_limited_api = True
+
+    if use_limited_api:
+        log.info(
+            'Targeting PEP 384 limited API supporting Python >= %d.%d',
+            *version_info)
+        version_hex = _version_info_to_version_hex(*version_info)
+        for ext in ext_modules:
+            ext.py_limited_api = True
+            ext.define_macros.append(('Py_LIMITED_API', version_hex))
```

Comment:
(sorry for my ignorance on this one)

Is this just injecting a variable so that extensions can read it? If so we should document that well, so people know they can rely on it? Or is this a normal thing that's available we have to do something special for?

Reply:
This macro must be set when you `#include <Python.h>` so that it limits what symbols are defined.